web/src/i18n/locales/en/errors.yaml

```diff
@@ -0,0 +1,17 @@
+# Default error messages.  These can be overridden by "families" of errors, but
+# if a "family" doesn't define a message, it will fallback to these up here.
+email-exists: Another account already exists with that email address
+invalid-phone: Phone number may not be more than 10 digits
+weak-password: The provided password is too weak
+
+# Errors specific to adding a new account
+add-account:
+  invalid: Email and password are required
+
+# Errors specific to editing an existing account
+edit-account:
+  invalid-state: The state selected for the account is invalid
+  invalid-role: The authorization role selected for the account is invalid
+
+# Errors specific to editing your own account
+edit-self:
```

Comment:
Tagging @jameshupp to get eyes on this.  This is an initial proposal from dev for how error messages might be structured in yaml.  The idea is there are some defaults (e.g., if every weak password error should display the same way, we only need to have it in the yaml once), and then the ability to override on a "family" basis - so the `add-account` family of messages could override the weak password message if that was appropriate.

What do you think?

Reply:
Seems like the effect of this is that any particular string of text will exist in as few places as possible, and I can't think of anything more important that would override that. So yes! This is good!